src/main/java/org/mitre/synthea/world/agents/Provider.java

```diff
@@ -35,7 +41,7 @@
 
   // ArrayList of all providers imported
   private static ArrayList<Provider> providerList = new ArrayList<Provider>();
-  private static QuadTree providerMap = new QuadTree(500, 500); // node capacity, depth
+  private static QuadTree providerMap = new QuadTree(1400, 1400); // node capacity, depth
```

Comment:
I know the initial number and this new number are likely both pulled from a hat, but could you test out how much this impacts memory usage when full? We don't need a ton of detail but a general sense would be good. We can talk about how to do that.

Reply:
The original number was quite pulled from a hat, although it looks nicely rounded. It started out smaller until it ran out of space loading the providers. I slowly increased the size until it safely was able to load the biggest states: e.g. California, Texas.